falco/templates/pod-template.tpl

```diff
@@ -134,6 +134,10 @@ spec:
           {{- end }}
       {{- end }}
       volumeMounts:
+        - mountPath: /etc/falco/falcoctl-rules
```

Comment:
This differs from the default Falco configuration. How do we enforce this in the Falco config?